### PR TITLE
Monitor filter updates

### PR DESCRIPTION
**Pull request by dmitryduev**
In this PR: 

- `dask` workers will now start a thread that would periodically reload user-defined filters.
- change some wordings in `readme.md`

### CODE REVIEW

File: kowalski/alert_broker_ztf.py [hunk 1]
```diff
@@ -849,11 +850,70 @@ def __init__(self, **kwargs):
...
+        if method == "get":
+            response = methods[method](
+                f"{config['skyportal']['protocol']}://"
+                f"{config['skyportal']['host']}:{config['skyportal']['port']}"
+                f"{endpoint}",
+                params=data,
+                headers=self.session_headers,
+            )
+        else:
+            response = methods[method.lower()](
```
Comment (kmshin1397) [inline kowalski/alert_broker_ztf.py:895]:
```suggestion
            response = methods[method](
```
already lowercased earlier in the function

**Review by kmshin1397 — COMMENTED** [review]
One small change

File: kowalski/alert_broker_ztf.py [hunk 2]
```diff
@@ -935,52 +998,18 @@ def __init__(self, **kwargs):
...
-                f"{config['skyportal']['protocol']}://"
-                f"{config['skyportal']['host']}:{config['skyportal']['port']}"
-                f"{endpoint}",
-                json=data,
-                headers=self.session_headers,
-            )
-
-        return response
+            # log("Re-loaded user-defined filters:")
+            # log(self.filter_templates)
```
Comment (kmshin1397) [inline kowalski/alert_broker_ztf.py:1012]: Commented out lines?

File: kowalski/alert_broker_ztf.py [hunk 3]
```diff
@@ -935,52 +998,15 @@ def __init__(self, **kwargs):
...
-        else:
-            response = methods[method.lower()](
-                f"{config['skyportal']['protocol']}://"
-                f"{config['skyportal']['host']}:{config['skyportal']['port']}"
-                f"{endpoint}",
-                json=data,
-                headers=self.session_headers,
-            )
+    def reload_filters(self):
+        while True:
```
Comment (stefanv) [inline kowalski/alert_broker_ztf.py:1005]: Consider `threading.Timer`: https://docs.python.org/3/library/threading.html#timer-objects

File: kowalski/alert_broker_ztf.py (hunk 2 above)
Comment (dmitryduev) [reply kowalski/alert_broker_ztf.py]: sorry, rm'd

**Review by kmshin1397 — APPROVED** [review]

**Review by stefanv — APPROVED** [review]
Looks good; minor suggestion that you can take or leave.

**Pull request merged**